src/wp-includes/load.php

```diff
@@ -162,10 +162,11 @@ function wp_check_php_mysql_versions() {
 		exit( 1 );
 	}
 
+	// This runs before default constants are defined, so we can't assume WP_CONTENT_DIR is set yet.
+	$wp_content_dir = defined( 'WP_CONTENT_DIR' ) ? WP_CONTENT_DIR : 'wp-content';
+
 	if ( ! function_exists( 'mysqli_connect' ) && ! function_exists( 'mysql_connect' )
-		// This runs before default constants are defined, so we can't assume WP_CONTENT_DIR is set yet.
-		&& ( defined( 'WP_CONTENT_DIR' ) && ! file_exists( WP_CONTENT_DIR . '/db.php' )
-			|| ! file_exists( ABSPATH . 'wp-content/db.php' ) )
+		&& ! file_exists( ABSPATH . $wp_content_dir . '/db.php' )
```

Comment:
```suggestion
		&& ! file_exists( $wp_content_dir . '/db.php' )
```

Hi @SergeyBiryukov I just did a quick review and I think `ABSPATH` is unnecessary here since `WP_CONTENT_DIR` already contains `ABSPATH` by `wp_initial_constants()`.

Reply:
Ah yes, you're absolutely right. Thanks! Fixed 🙂